main.go

```diff
@@ -81,6 +82,14 @@ func main() {
 	// Create the context.
 	ctx := context.Background()
 
+	log.Info("Ensuring Calico datastore is initialized")
+	initCtx, cancelInit := context.WithTimeout(ctx, 10*time.Second)
+	defer cancelInit()
+	err = calicoClient.EnsureInitialized(initCtx, "", "k8s")
```

Comment:
hmm, I think we probably want to leave the clusterType blank here. They get configured in the install manifests.

Reply:
ClusterType is cumulative though, is it worth having something in there that pops up whenever kube-controller is deployed?

Reply:
I think we agreed that we were all OK with `k8s` offline.  It acts as a fallback if the user isn't using our manifest for whatever reason.